includes/Services/FlowService.php

```diff
@@ -0,0 +1,262 @@
+<?php
+namespace NewfoldLabs\WP\Module\Onboarding\Services;
+
+use NewfoldLabs\WP\Module\Onboarding\Data\Options;
+use NewfoldLabs\WP\Module\Onboarding\Data\Flows\Flows;
+use NewfoldLabs\WP\Module\Onboarding\Data\Data;
+
+/**
+ * Class FlowService
+ */
+class FlowService {
+
+	/**
+	 * Initialize Flow Data on refresh.
+	 *
+	 * @return boolean
+	 */
+	public static function initalize_flow_data() {
+		$default_flow_data = self::get_default_flow_data();
+		$flow_data         = self::read_flow_data_from_wp_option();
+
+		if ( ! ( $flow_data ) ) {
+			return \update_option( Options::get_option_name( 'flow' ), $default_flow_data );
+		}
+
+		if ( ! isset( $flow_data['version'] ) || strcmp( $flow_data['version'], $default_flow_data['version'] ) !== 0 ) {
+			$flow_data_upgrades = NFD_ONBOARDING_DIR . '\\includes\\Data\\Flows\\Upgrades\\' . $default_flow_data['version'] . '.php';
+			if ( $flow_data_upgrades ) {
+				require $flow_data_upgrades;
```

Comment:
Should we user require or include? cause if require fails it returns a fatal error and the application might crash.
https://www.w3schools.com/php/php_includes.asp#:~:text=The%20include%20and,script%20will%20continue

Reply:
Yes agreed, considering that while writing to the updated version files, we can avoid such fatal errors, hence included `require`. However, to avoid such scenarios where the site might crash, shall revert it to `includes`